test/integration/targets/filter_urls/runme.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: testhost
+  gather_facts: True
+  tasks:
+    - name: Test urldecode filter
```

Comment:
It looks like there are basically 3 copies of nearly the same test tasks. This file, the ones in `jinja26.yml` and then the original tests in `tasks/main.yml`. Would it be possible to use just a single playbook and run it twice, once using the venv and once without? The ones in `jinja26.yml` appear the most complete.

Reply:
The one in `tasks` was meant to be nuked, but yes they should be able to be combined into one playbook.